.github/workflows/coveralls.yml

```
@@ -0,0 +1,27 @@
+name: Coveralls
+
+on: ["push", "pull_request"]
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v1
+
+    - name: Use Node.js LTS 14.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+
+    - name: Install dependencies and execute tests
+      run: |
+        npm ci
+        npm run coverage
+
+    - name: Coveralls
+      uses: coverallsapp/github-action@master
```

I'd prefer to use a fixed version
```suggestion
      uses: coverallsapp/github-action@v1.1.2
```

---

Done